Parser/readWQMraw.m

```diff
@@ -620,8 +620,8 @@
     params{end + 1} = {'DO(mmol/m^3)', {'DOX1', ''}}; % mmol/m3 <=> umol/l
     params{end + 1} = {'oxygen', {'DOX', ''}};
     params{end + 1} = {'fluorescence', {'FLU2', ''}};
-    params{end + 1} = {'F_Cal_CHL', {'CHPL', getCPHLcomment('factory','470nm','695nm')}};
-    params{end + 1} = {'U_Cal_CHL', {'CHPL', getCPHLcomment('user','470nm','685nm')}};
+    params{end + 1} = {'F_Cal_CHL', {'CHPL', getCPHLcomment('factory', '470nm', '695nm')}};
```

Comment:
CHPL here!!! Should be CPHL.

Reply:
Ahh, again, the same thing - this came through WQM merge at wrong place.